gatorgrouper/utils/gatorgrouper.py

```diff
@@ -14,8 +14,12 @@
 
 if __name__ == "__main__":  # pragma: no cover
 
-    # parse the arguments and display welcome message
+    # parse the arguments
     GG_ARGUMENTS = parse_arguments.parse_arguments(sys.argv[1:])
+    if (GG_ARGUMENTS == "Invalid entry"):
```

Comment:
This is not checking for the right string -- the string returned should be declared in `constants.py`.